### PR TITLE
#611 update mutation config

### PR DESCRIPTION
**Pull request by hsimah**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------
Creates public functions for each mutation class's configuration of input, output and mutation. I wanted to mutate a CPT with custom fields while retaining all the base fields from `PostTypeCreate`. I went through and refactored all mutations (except password recovery email, seemed superfluous) to be consistent.

Does this close any currently open issues?
------------------------------------------
#611 

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
I haven't rigorously tested this. It works in de facto use in my app (I have been using my fork in production for several weeks). I would hope the team could run the automated tests in the project - I have no idea how to do that!

Where has this been tested?
---------------------------
**Operating System:** Windows 10, Red Hat

**WordPress Version:** 5.0.2, latest 4.x (prior to 5 release).

### CODE REVIEW

**Comment by hsimah** [comment]
@jasonbahl yeah, I accepted a merge conflict without checking too carefully. I have (hopefully) fixed it. We'll see how this build goes.

I think I will need a crash course in running tests!

File: src/Mutation/MediaItemUpdate.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public static function get_output_fields() {
 	}
 
 	public static function mutate_and_get_payload() {
-		function ( $input, AppContext $context, ResolveInfo $info ) {
+		return function ( $input, AppContext $context, ResolveInfo $info ) {
```
Comment (jasonbahl) [inline src/Mutation/MediaItemUpdate.php]: @hsimah looks like this just wasn't being returned. Made the change and confirmed all tests pass locally. 

I'll review the code, but on initial scan I'm digging the change

Comment (hsimah) [reply src/Mutation/MediaItemUpdate.php]: d'oh!

**Comment by hsimah** [comment]
@jasonbahl docblocks added :)

**Pull request merged**